sdk/storage/azure-storage-common/src/storage_switch_to_secondary_policy.cpp

```diff
@@ -13,10 +13,7 @@ namespace Azure { namespace Storage { namespace _internal {
       const Azure::Core::Context& ctx) const
   {
     std::shared_ptr<bool> replicaStatus;
-    if (ctx.HasKey(SecondaryHostReplicaStatusKey))
-    {
-      replicaStatus = ctx.GetValue<std::shared_ptr<bool>>(SecondaryHostReplicaStatusKey);
-    }
+    ctx.TryGetValue<std::shared_ptr<bool>>(SecondaryHostReplicaStatusKey, replicaStatus);
```

Comment:
If it is possible to invoke without explicitly putting `<std::shared_ptr<bool>>`, it is better

Reply:
From the existing code I read, it wasn't possible with `GetValue`, so why would it be possible now with `TryGetValue`?

I will try it and see.

Reply:
It works without it. Leaving it as is, since the existing code was explicit.